### DISCUSSION TITLE
Can we add Aarhus, Denmark?

### DISCUSSION THREAD

**Discussion by Godtfolk** [mampfes/hacs_waste_collection_schedule#2228] [Ideas]
Can we add Aarhus, Denmark?

Waste and recycling in Aarhus, Denmark area 

https://www.kredslob.dk/produkter-og-services/genbrug-og-affald/affaldsbeholdere/toemmekalender

Example adress: 

Skovparken 5, 8240 Risskov 

= https://portal-api.kredslob.dk/api/calendar/icsfeed/07517381___5_______/67799

Credits: https://github.com/kvedes/hass-aarhus-waste

**Comment by 5ila5** [comment]
kredslob.dk is supported by the [ICS source ](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/source/ics.md) I added dedicated documentation in #2229 you can find it [here](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/ics/kredslob_dk.md)
